src/config/options.rs

```diff
@@ -49,6 +49,13 @@ pub enum ControlBraceStyle {
     AlwaysNextLine,
 }
 
+#[config_type]
+pub enum TrailingSemicolon {
+    Always,
+    Never,
+    Preserve,
+}
```

Comment:
nit: Although I think it's self explanatory, let's add some doc comments like the other options